src/styles.css

```
@@ -202,6 +202,65 @@ body{background:var(--bg);color:var(--ink);font-family:'Plus Jakarta Sans',sans-
 /* ── GRID ── */
 .grid{display:grid;grid-template-columns:repeat(auto-fill,minmax(310px,1fr));gap:12px;margin-bottom:60px}
 
+/* ── SCROLLABLE DIRECTORIES ── */
+#orgGrid {
```

Stylelint (17.14.0)

[error] Expected id selector "`#orgGrid`" to be kebab-case (selector-id-pattern)

(selector-id-pattern)